features/invitelist/impl/src/main/kotlin/io/element/android/features/invitelist/impl/InviteListPresenter.kt

```diff
@@ -95,8 +106,17 @@ class InviteListPresenter @Inject constructor(
             }
         }
 
+        val inviteList by remember(seenInvites) {
+            derivedStateOf {
+                invites
+                    .filterIsInstance<RoomSummary.Filled>()
+                    .map { it.toInviteSummary(seenInvites.value.contains(it.details.roomId)) }
```

Comment:
We could use some smart mechanism with the `key` composable method to avoiding remapping everything. But thats optimisation.

Reply:
Could do, but feels a bit premature? I can't see the invite list changing that frequently that this becomes a problem?